samples/f27-build-from-ubuntu1804.json

```diff
@@ -1,29 +0,0 @@
-{
-  "pipeline": {
-    "build": {
-      "runner": "org.osbuild.ubuntu1804"
-    },
-    "stages": [
-      {
-        "name": "org.osbuild.yum",
-        "options": {
-          "releasever": "27",
-          "basearch": "x86_64",
-          "repos": [
-            {
-              "baseurl": "https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/$releasever/Everything/$basearch/os/",
-              "gpgkey": "-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nmQINBFiskqMBEADTbsoAXpDPk+FtcwBEPZQVe0YQYdOqavfQQVD/RYAcHnJW/K1b\nZhQusBjUIec9SfGi3uBNNmbziAvpd/ycMKyWHuWQLmBgbImrqnPbBPMXmxeNGnZj\nA1hjVDp0pzj2+gZQhqYWSf6kQy9u9A1mSU63Kl/tfw7+hX7Tc3I8feGAFCHcFQgE\nSxUib8Mw/OOGR3Am9fKdA+K1kJeQIiZvXMcNFx+3CfoavhFdicuoT2KbcSuzRm76\nduKNHlLaP6/IbZxNiDWh8SDVpFaFPlqR/R/+wibA6e9wMf6CZ4vfUY7NKYf4tYBs\n0EYdkn3j/KhJJxdb+M46Q/xwq9ovZo7XIhLrIUPuMw91X9cbvkU/a9kE1ffdpNmF\n1fDnUcEkuuEqOl+aMVsUBEbAQ86yrwpDfL4XT9vwnDIkggKZyvDTZ6q00XKg3Ger\nKuZtQBl/YcHDXuBlB1fzpGl8a8hq/+GeT2sVxjlYwPXjrsKd1NeP6ctQsR6gHuP3\nW5ohP4rArtM6ONN8rlTiodDLVGHBpUzIRdgr7RCL1AqB9vrdQ2MVZMasTcUnUvKo\n0H4mps+x05jGao0b0Z0TJc6wr6ybHH38NVG06VX5rJZlfZchGwkWzmYxai55/7ln\n1vJmk+kbdS7pK0jfmeVJ8A77XLCL36oJFCiCrYjV+ZGgvB+z08Jzwc7sgwARAQAB\ntCxGZWRvcmEgMjcgKDI3KSA8ZmVkb3JhLTI3QGZlZG9yYXByb2plY3Qub3JnPokC\nOAQTAQIAIgUCWKySowIbDwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AACgkQ9V50\nMPUoLuTcww/+LFPVEyVguMeU/QABEsE5FEN7kcDReZtdwq7p/aKC29mzCxeHggit\nYOGlrINkJ26Aq6p+oW6w7JxBWJnKoTBYJDFzNIbp/6GbG4oKcEnWQZfTnRLTr5au\nkVdWBFevzC0huraobKz3joYRIX826VUzS/A418zpnDVPtpo3x86V9f292rqi2tn9\nQ5cQC5Ck3/cjQDEwkN9gHz4j/c1oa6zBOcHbKkaZdWA2dIs6XOxCIHg78i6VQwMM\ns+vfm2vbV3ACCcOVnd3d6NxIQuDLEQwdtdB2zI8R74bjacosrcafK+F2DnkM7WrL\nSCiTKLJBMRDx+X2nOjT5pLuts4FC/XYRO23SMtPAMzQ852Z1lkjsaVDsjzNqCasU\nB0vDPHLQE8aj0zchNBSzuHoKpXNYTyJztekWL/QXkUsXu0x7N5WhBlZ+lni6LtZU\n51l7BJd1n+ZKnQ4gmjQ1ffVLbgzb9Z1MNje0s61FdKmUJQGULYqh32W4GV+RLvtI\n6AJV/EmFCUEfRJ3eA8tJiyKe512wiim/WDhvzFuuPBup2Z3TufiaJQOysLlN5+HU\nQitTtcd7j8ZgpsIAIZtSWOMxbIAJWbzn8gjfIj4ZDeo0ZZXH0VgDkXtpv1g8R2aR\nAz6ob5YYW5VnI2UEr53x1Z7lmUhv/TUZn26IeU16jCJ80k7pJvQSF8Y=\n=xpp1\n-----END PGP PUBLIC KEY BLOCK-----\n"
-            }
-          ],
-          "packages": [
-            "dnf",
-            "systemd",
-            "tar",
-            "gnupg"
-          ]
-        }
-      }
-    ]
-  },
-  "runner": "org.osbuild.fedora27"
```

Comment:
This was the only place we're using the `fedora27` runner, right? Should we remove it?

Reply:
Yeah, I think so. Probably we can delete a lot after this.